src/SecureHeadersHandler.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace SecureHeaders\PsrHttpAdapter;
+
+use Aidantwoods\SecureHeaders\SecureHeaders;
+use Psr\Http\Message\RequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+
+/**
+ * Secure headers handler
+ *
+ * Middleware to apply Secure headers to a PSR7 response
+ */
+class SecureHeadersHandler
```

Comment:
Just reflecting on https://github.com/aidantwoods/SecureHeaders/pull/64#issuecomment-334381463, should this be perhaps called something like `GenericHandler` to reflect the common pattern, and then when PSR-15 arrives we can also seperately introduce a `Psr15Handler`. (The `SecureHeaders` prefix is probably unneeded because of namespacing?)

Reply:
What about `ApplySecureHeaders`? That would be a nice name to read when building a pipeline of middlewares. Separating PSR-7 and PSR-15 implementations could be done by using a different namespace?

Reply:
I think `ApplySecureHeaders` seems nice. I'd think that once psr-15 is official, the psr-15 version would *replace* the the generic implementation, and there'd be a version bump..

Reply:
I think we should just provide both once PSR-15 comes around.

Reply:
I think both is probably best. In which case, we should perhaps namespace this ahead of time

Reply:
On reflection, I think we can probably just name the Psr15 implementation `Psr15Handler` or something similar, doesn't seem worth namespacing a single file – as long as it is clear.